includes/_header.htm

```diff
@@ -56,4 +57,17 @@
 				}
 			}
 		</script>
+
+
+			<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+			<script type="text/javascript">
+				document.addEventListener("DOMContentLoaded", function(event) {
+					docsearch({
+					apiKey: 'aa09166cb320ff3715048b35f14631ca',
+					indexName: 'kafka_apache',
+					inputSelector: '#searchbox',
+					debug: true // Set debug to true if you want to inspect the dropdown
```

Comment:
@zer0Id0l You can set this boolean to false in order to automatically hide the dropdown. It will stay open otherwise.

Reply:
Made the changes. Thanks!